src/App.php

```diff
@@ -1128,7 +1132,11 @@ protected function outputResponse(string $data): void
             return;
         }
 
-        $this->response->getBody()->write($data);
+        $stream = $this->response->getBody();
+        if ($data !== '' && $stream->isWritable()) {
```

Comment:
the 2nd cond should not be needed as can loose data if `$data !== ''`

Reply:
ok will try